evennia/contrib/actions/__init__.py

```diff
@@ -0,0 +1,43 @@
+from evennia import ObjectDB, DefaultCharacter, DefaultRoom
+from evennia.contrib.actions.actions import Action
+from evennia.contrib.actions.handlers import CharacterActionHandler, RoomActionHandler
+from evennia.contrib.actions.scripts import ActionSystemScript
+from evennia.contrib.actions.typeclasses import ActionCharacter, ActionRoom, ActionExit
+from evennia.contrib.actions.commands import ActionCmdSet 
+from evennia.contrib.actions.debug import ActionDebugCmdSet
+from evennia.contrib.actions.setup import setup
+
+__all__ = [Action, ActionCharacter, ActionRoom, ActionExit, ActionCmdSet, 
+    ActionDebugCmdSet, setup]
+
+
+def setup(override=False):
```

Comment:
This code should not be repeated both here and in setup.py. I suggest moving this functioninto the utils.py module instead and removing setup.py completely.